src/generators.rs

```diff
@@ -69,12 +71,19 @@ where
 ///   image
 ///
 #[pyfunction]
-#[text_signature = "(/, num_nodes=None, weights=None)"]
+#[text_signature = "(/, num_nodes=None, weights=None, bidirectional=False)"]
 pub fn directed_cycle_graph(
     py: Python,
     num_nodes: Option<usize>,
     weights: Option<Vec<PyObject>>,
+    bidirectional: Option<bool>,
```

Comment:
This is a bit redundant since we're basically saying this can be None (which is treated as false) or true or false. I think when you couple this with my comment above above about the macro you can just set this to be a `bool`